File: java/perimeter.java
/* Perimeter of the rectangle*/
import java.io.*;
class perimeter
{
	public static void main(String args[])throws IOException
	{
		int length,breadth,perimeter;
		DataInputStream dis=new DataInputStream(System.in);
		System.out.println("Enter the length of the rectangle");
		length=Integer.parseInt(dis.readLine());
		System.out.println("Enter the breadth of the rectangle");
		breadth=Integer.parseInt(dis.readLine());
		perimeter=(2*length)+(2*breadth);
		System.out.println("The perimeter of the rectangle "+perimeter);
	}
}

File: c/perimeter.c
/*Perimeter of the rectangle*/

#include<stdio.h>
int main()
{
	int length,breadth,perimeter;
	printf("Enter the length of the rectangle \n");
	scanf("%d",&length);
	printf("Enter the breadth of the rectangle \n");
	scanf("%d",&breadth);
	perimeter=(2*length)+(2*breadth);
	printf("The area of the rectangle is %d \n",perimeter);
}